lib/User/UserEntry.php

```diff
@@ -163,9 +164,6 @@ public function getUUID() {
 				$this->connection->ldapExpertUUIDUserAttr = $uuidAttribute;
 				$this->connection->saveConfiguration(); // FIXME should not be done here. Move to wizard?
 			}
-			if ($uuidAttribute === 'objectguid' || $uuidAttribute === 'guid') {
```

Comment:
This shouldn't be needed since the value should be transformed in the `getAttributeValue` call. The change in the `getAttributeValue` was added in https://github.com/owncloud/user_ldap/pull/711 due to some problems